api/v3/openapi.yaml

```diff
@@ -898,6 +898,29 @@ paths:
         '5XX':
           $ref: '#/components/responses/ServerErrorResponse'
 
+    patch:
+      summary: 'Update partition count'
+      operationId: updatePartitionCountKafkaV3Topic
+      description: |-
+        [![Generally Available](https://img.shields.io/badge/Lifecycle%20Stage-Generally%20Available-%2345c6e8)](#section/Versioning/API-Lifecycle-Policy)
+
+        Increases number of partitions on a topic.  
+      tags:
+        - Topic (v3)
+      requestBody:
+        $ref: '#/components/requestBodies/UpdatePartitionCountRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/GetTopicResponse'
+        '400':
```

Comment:
Q: no-where do i see, these error-response code being generated in the code below?
How is exception handling done? What if admin-client throws?
PS: I vaguely remember from one of discussions before, that there is some common rest-utils/kafka-rest code to handle exception-handling <-> error code discussion.

Reply:
Yes, it's handled at the rest-utils/jetty layer

eg
```
 curl -X PATCH http://localhost:8082/v3/clusters/3W-2oFuqSYuErBzrRrTKyw/topics/bob  -H "Content-Type: application/json" -d '{"partition_count123":"16"}'  | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   175    0   148  100    27    701    127 --:--:-- --:--:-- --:--:--   829
{
  "error_code": 400,
  "message": "Cannot construct instance of `io.confluent.kafkarest.entities.v3.PartitionCountRequest`, problem: Null partitionCount"
```

429 comes from the jetty dos rate limiter/filter or from the Guava/Resiliance4J rate limiters

401 comes from kafka returning a not authed and an exception handler translates this to a 401

And unexpected stuff gets translated by an exception handler to 500 errors.